services/address_sanitizer.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import dataclass
+from multiprocessing.connection import Client
+from typing import Callable, List, Mapping, Set
+
+import googlemaps
+
+import googlemaps.places
+
+from config.settings import Settings
+from models.point_of_interest import PointOfInterest
+
+
+def make_places_request(address: str, gmaps: googlemaps.places):
+    return gmaps.places_autocomplete(address)
+
+# is there somewhere I can put this method to reuse it throughout the app?
```

Comment:
how about `services.google_utils`?

Reply:
let's make another NICE TO HAVE Ticket as this will require a decent amount of refactor and is not really within the scope.